src/nvim/window.c

```diff
@@ -700,26 +700,26 @@ void win_set_minimal_style(win_T *wp)
   }
 }
 
-void win_config_float(win_T *wp, FloatConfig fconfig)
+void win_config_float(win_T *wp, FloatConfig *fconfig)
```

Comment:
This changes the semantics.  Previously, `fconfg` was copied so any changes in this function don't affect the caller.  Now, the changes do affect the caller's config.  Is that an issue?

Reply:
Damn. Thanks for the heads up. I'm unsure, will need to investigate a bit.